.github/workflows/main-build.yml

```diff
@@ -87,12 +89,18 @@ jobs:
           build-args: "ADOT_JAVA_VERSION=${{ env.ADOT_JAVA_VERSION }}"
           context: .
           platforms: linux/amd64,linux/arm64
+          tags: ${{ env.TEST_TAG }}
+
+      - name: Test docker image
+        shell: bash
+        run: github/scripts/test-adot-javaagent-image.sh "${{ env.TEST_TAG }}"
```

Comment:
should be `.github/scripts/test-adot-javaagent-image.sh "${{ env.TEST_TAG }}"`

can you fix this in all workflows?